src/modules/course/docUtils/course.responsedoc.ts

```diff
@@ -195,6 +201,13 @@ export class ScheduleResponseBody {
    */
 
   time: string;
+
+  /**
+   * The lecture Number/Index among all lectures in the schedule
+   * @example 8
+   */
+
+  lectureNumber: number;
```

Comment:
What is this?

Reply:
missing field frim courseDetails.txt

Reply:
Videosnum is not sufficient?

Reply:
this is for the schedule
the index for every lecture

Reply:
How a number can store index of all the scheduled courses? @Devesh21700Kumar

Reply:
> How a number can store index of all the scheduled courses? @Devesh21700Kumar

Its a field of schedule schema so every schedule will store its *own* index only.

One object will have 1
another will have 2 
another object will have 3